jobs/BnRStatusPollerJob.js

```diff
@@ -147,7 +148,17 @@ class BnRStatusPollerJob extends BaseJob {
         }
       })
       .then(operationStatusResponse => operationStatusResponse.operationFinished ?
-        this.doPostFinishOperation(operationStatusResponse, operationName, instanceInfo) : operationStatusResponse
+        this.doPostFinishOperation(operationStatusResponse, operationName, instanceInfo)
+        .tap(() => {
+          if (operationStatusResponse.state === CONST.OPERATION.FAILED) {
+            const options = {
+              instance_id: instance_guid,
+              repeatInterval: utils.getCronWithIntervalAndAfterXminute(plan.service.backup_interval, CONST.SCHEDULE.RETRY_DELAY),
+              type: CONST.BACKUP.TYPE.ONLINE
+            };
+            return cf.serviceFabrikClient.scheduleBackup(options);
```

Comment:
can you do this in a retry block here. Just in case you cannot reschedule, then base backup wont be trigerred.

Reply:
also why is `serviceFabrikClient` added under cf module? it should be out on its own right? Its a data-access-layer on its own. This is not a review comment to you, but for SF 2.0.
@subhankarc @amitmalav  - comments?

Reply:
Retry logic is implicit here. Reason is if this fails will be rescheduled after 30 mins. If fails again check would be done by BnRStatuspoller and  will be rescheduled after 30 mins.

Reply:
@jagadish-kb makes sense to move it out of cf.

Reply:
Added retry logic for schedule backup to retry even if the `scheduleBackup` call fails.